db/migrate/20200205154839_update_vssc_extract_date_value.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateVsscExtractDateValue < ActiveRecord::Migration[5.2]
+    def up
+      DrivetimeBand.update_all("vssc_extract_date='2019-10-31 00:00:00'")
```

Comment:
Any particular reason to use a string argument instead of a hash?
```suggestion
      DrivetimeBand.update_all(vssc_extract_date: '2019-10-31 00:00:00')
```

Reply:
No reason. I was just thinking in SQL.